harvard-lts/etd-integration-tests
# Etd 241

**Enhance itest for dash.**
* * *

**JIRA Ticket**: [(link)](https://jira.huit.harvard.edu/browse/ETD-241)

# What does this Pull Request do?
This pr inserts a test object to dash, and attempts another to test duplicate detection, and cleans up after test by deleting test object from dash and the filesys. As of now we plan to do this on dev/qa, but on prod, we will only pass a hello world message

# How should this be tested?

Build locally to confirm build
Deploy to dev along with IF, and run itest (done)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests? This is the integration test

# Interested parties
Tag (@ mention) interested parties
